crates/hc_sandbox/tests/cli.rs

```diff
@@ -433,6 +433,18 @@ async fn get_launch_info(mut child: tokio::process::Child) -> LaunchInfo {
             return serde_json::from_str::<LaunchInfo>(launch_info_str).unwrap();
         }
     }
+
+    let mut buf = String::new();
+    if let Some(stderr) = child.stderr.take() {
+        BufReader::new(stderr)
+            .read_to_string(&mut buf)
+            .await
+            .unwrap();
+        eprintln!("{buf}");
+    } else {
+        panic!("No stderr! Was the process handle dropped?");
+    }
+
```

Comment:
`stderr` is inherited wherever `get_launch_info` is used. No need to write custom code to just output errors. Plus this has nothing to do with tx2 removal.
```suggestion
```

Reply:
@jost-s - pretty sure the stderr in the child is being set to pipe, otherwise this would panic with the No stderr panic. Thus we need to manually print stderr to see it. But I also don't care, so don't mind that you've removed this change.

Reply:
No, I checked again before reverting. I know about this because when I fixed the dpki flag in hc sandbox, I changed them all to inherit stderr.